yarn.lock

```diff
@@ -3830,31 +3830,6 @@
   resolved "https://registry.yarnpkg.com/@protobufjs/utf8/-/utf8-1.1.0.tgz#a777360b5b39a1a2e5106f8e858f2fd2d060c570"
   integrity sha1-p3c2C1s5oaLlEG+OhY8v0tBgxXA=
 
-"@sendgrid/client@^6.5.5":
```

Comment:
what's happening here? I dont see any modifications to the `package.json`

Reply:
I just ran yarn and got that. My guess is that a previous PR didn't update the lock file? I was confused too, so I discarded the changes and ran yarn again and it produced the same thing.